modules/introduction/partials/new-features-80.adoc

```diff
@@ -35,6 +34,10 @@ The following settings have been added to the `/pools/default/buckets` REST APIs
 |51 to 90
 |85
 |===
++
+The settings `warmup_min_items_threshold`, `warmup_min_memory_threshold`, `exp_pager_stime`, `mem_high_wat`, and `mem_low_wat` are deprecated, and replaced by the above new settings.
++
+For more information, see the xref:rest-api:rest-bucket-create.adoc[Creating and Editing Buckets] and xref:learn:buckets-memory-and-storage/memory.adoc[Memory].
```

Comment:
I think here it would be nice if we referenced the MB-9418 entry further down the page (and vice-versa).